tools/release/release-output/output-validations.ts

```diff
@@ -125,11 +124,12 @@ export function checkPrimaryPackageJson(
   } else if (packageJson.version !== expectedVersion) {
     failures.push(
         `Unexpected package version. Expected: ${expectedVersion} but got: ${packageJson.version}`);
-  }
-
-  if (packageJson['ng-update'] && packageJson['ng-update'].migrations) {
+  } else if (semver.valid(expectedVersion) === null) {
+    failures.push(`Version is not satisfying SemVer specification: ${packageJson.version}`);
```

Comment:
This might read slightly better as

`Version does not satisfy SemVer specification: ${packageJson.version}`

Reply:
Yeah, agreed. Thanks for raising this!